.github/workflows/ci.yml

```diff
@@ -22,18 +25,20 @@ jobs :
         kotlin-version : [ 1.8.0 ]
 
     steps :
-      - uses : actions/checkout@v2
-      - uses : actions/setup-java@v2
+      - uses : actions/checkout@v3
+      - uses : actions/setup-java@v3
         with :
           distribution : 'adopt'
           java-version : '11'
           check-latest : true
 
       - name : Test on Ubuntu
-        run : ./gradlew assemble test -Psquare.useIR=false --no-build-cache --no-daemon --stacktrace -Psquare.kotlinVersion=${{ matrix.kotlin-version }}
+        env:
+          DEP_OVERRIDE_config-useIR: "false"
```

Comment:
How does this work now when running from commandline locally? Same with e.g. wanting to run a command with a different version of Kotlin

Reply:
you can either just manually change it in `libs.versions.toml` or do `DEP_OVERRIDES=true DEP_OVERRIDE_kotlin=1.8.10-beta ./gradlew <task>`

Reply:
What exactly is the command? I frequently disable running all tests either through my `gradle.properties` in my home dir `square.fullTestRun=false` or as `-Psquare.fullTestRun=false`. I can't follow what I do need to do in both cases now.

Reply:
instead of `-Psquare.fullTestRun=false` you would prefix your command with this

```
DEP_OVERRIDES=true DEP_OVERRIDE_config-fullTestRun=true ./gradlew <your check>
```

`DEP_OVERRIDES` is a flag to inform dependency management that we have overrides in env vars it should look for. `DEP_OVERRIDE_<toml version key>` is the syntax for replacing a toml version key with something else.